test/e2e/alerts-existing-cozy.js

```diff
@@ -0,0 +1,223 @@
+  parser.addArgument(['-v', '--verbose'], { action: 'storeTrue' })
+  return parser.parseArgs()
+}
+
+const ruleFormatter = formatAccountOrGroup => {
+  return rule => {
+    return `Rule<${rule.value} for ${formatAccountOrGroup(
+      rule.accountOrGroup
+    )}>`
+  }
+}
+
+const accountOrGroupFormatter = (accountsById, groupsById) => {
+  return accountOrGroup => {
+    if (!accountOrGroup) {
+      return 'all accounts'
+    } else if (accountOrGroup._type === ACCOUNT_DOCTYPE) {
+      return (
+        getAccountLabel(accountsById[accountOrGroup._id]) || 'Unknown account'
+      )
+    } else if (accountOrGroup._type === GROUP_DOCTYPE) {
+      const fakeT = x => x
+      return (
+        getGroupLabel(groupsById[accountOrGroup._id], fakeT) || 'Unknown group'
+      )
+    } else {
+      return `Unknown<_type: ${accountOrGroup._type}, _id: ${
+        accountOrGroup._id
+      }>`
+    }
+  }
+}
+
+const isDesignDoc = x => x._id.startsWith('_design')
+const fetchData = async client => {
+  const { data: settings } = await client.collection(SETTINGS_DOCTYPE).getAll()
+  const { data: accounts } = await client.collection(ACCOUNT_DOCTYPE).getAll()
+  const { data: groups } = await client.collection(GROUP_DOCTYPE).getAll()
+  return {
+    settings: settings.filter(x => !isDesignDoc(x)),
+    accounts: accounts.filter(x => !isDesignDoc(x)),
+    groups: groups.filter(x => !isDesignDoc(x))
+  }
+}
+
+const formatTransaction = transaction => {
+  return `Transaction<${transaction.label}, ${transaction.amount}>`
+}
+
+const chooseAccount = async accounts => {
+  const prompt = inquirer.createPromptModule()
+  const answers = await prompt([
+    {
+      name: 'chosenAccount',
+      message: 'Create transaction on which account ?',
+      type: 'list',
+      choices: accounts.map(account => ({
+        value: account,
+        name: account.label
+      }))
+    }
+  ])
+  return answers.chosenAccount
+}
+
+const insertFakeDocuments = async client => {
+  const {
+    settings: allSettings,
+    accounts: allAccounts,
+    groups: allGroups
+  } = await fetchData(client)
+
+  const notificationSettings = allSettings.find(
+    setting => setting.notifications
+  )
+  const balanceLowerRules = notificationSettings.notifications.balanceLower
+
+  const accountsById = keyBy(allAccounts, getDocumentID)
+  const groupsById = keyBy(allGroups, getDocumentID)
+  const formatAccountOrGroup = accountOrGroupFormatter(accountsById, groupsById)
+  const formatRule = ruleFormatter(formatAccountOrGroup)
+
+  console.log('Existing rules:')
+  console.log(
+    balanceLowerRules
+      .map(formatRule)
+      .map(x => `- ${x}`)
+      .join('\n')
+  )
+
+  for (const rule of balanceLowerRules) {
+    const correspondingAccounts = allAccounts.filter(
+      ruleAccountFilter(rule, allGroups)
+    )
+    let chosenAccountId
+    if (correspondingAccounts.length > 1) {
+      console.log('More than 1 corrsponding account for rule', formatRule(rule))
```

Comment:
`console.log('More than 1 corresponding account for rule', formatRule(rule))` (missing "e")